scalalib/src/mill/scalalib/publish/LocalPublisher.scala

```diff
@@ -6,12 +6,13 @@ object LocalPublisher {
 
   private val root: Path = home / ".ivy2" / "local"
 
-  def publish(jar: Path,
-              sourcesJar: Path,
-              docsJar: Path,
-              pom: Path,
-              ivy: Path,
-              artifact: Artifact): Unit = {
+  def publish(
+      jar: Path,
+      sourcesJar: Path,
+      docsJar: Path,
+      pom: Path,
+      ivy: Path,
+      artifact: Artifact): Unit = {
```

Comment:
Can you configure Scalafmt to follow the IntelliJ convention of vertical-aligning multiline argument lists with the opening paren? This applies to both class definitions and functions. I believe there is a flag somewhere inside scalafmt that should do this easily